src/pages/talks/index.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react'
+import Link from 'next/link'
+import Image from 'next/image'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+
+const Talks: React.FC = ({allTalks}: any) => {
+  console.log({allTalks})
+
+  return (
+    <div className="mx-auto max-w-screen-lg lg:py-16 py-10">
+      <h1 className="md:text-4xl text-2xl text-center font-bold pb-16">
+        egghead Talks
+      </h1>
+      <div className="grid md:grid-cols-2 grid-cols-1 md:gap-16 gap-8">
+        {allTalks.map((talk: any) => {
+          return (
+            <div key={talk.path} className="flex flex-col">
+              {talk.image ? (
+                <div className="md:mb-4 mb-2">
+                  <Link href={talk.path}>
+                    <a>
+                      <Image
+                        src={talk.image}
+                        alt={talk.title}
+                        width={1280}
+                        height={720}
+                        quality={100}
+                        className="rounded-lg"
+                      />
+                    </a>
+                  </Link>
+                </div>
+              ) : (
+                <div className="aspect-w-16 aspect-h-9 md:mb-4 mb-2">
+                  <Link href={talk.path}>
+                    <a>
+                      <div className="absolute top-0 left-0 w-full h-full bg-gray-200 dark:bg-gray-800 rounded-lg flex items-center justify-center text-gray-400 dark:text-gray-600">
+                        <IconPlaceholder />
+                      </div>
+                    </a>
+                  </Link>
+                </div>
+              )}
+              <Link href={talk.path}>
+                <a>
+                  <h2 className="md:text-2xl text-xl font-bold leading-tighter">
+                    {talk.title}
+                  </h2>
```

Comment:
```suggestion
                  <h2 className="md:text-2xl text-xl font-bold leading-tighter hover:text-blue-600 dark:hover:text-blue-300">
                    {talk.title}
                  </h2>
```

I believe we want to keep the hover effect of the title changing to blue